src/gluonnlp/model/bert.py

```diff
@@ -581,6 +593,8 @@ def _bert_model(model_name=None, dataset_name=None, vocab=None, pretrained=True,
                           use_residual=predefined_args['use_residual'])
     # vocab
     vocab = _load_vocab(dataset_name, vocab, root)
```

Comment:
Instead of adding the from_vocab API, I suggest implement a _load_bert_vocab API which calls BERTVocab.from_json